src/App.jsx

```diff
@@ -1,104 +1,20 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+const posts = postsFromServer.map(post => ({
+  ...post,
+  user: usersFromServer.find(user => user.id === post.userId) || null,
+  comments:
+    commentsFromServer.filter(comment => comment.postId === post.id) || [],
```

Comment:
```suggestion
    commentsFromServer.filter(comment => comment.postId === post.id),
```
redundant here, 'cause filter method always returns the new array